extensions/sql-migration/src/wizard/assessmentDetailsPage/assessmentDetailsHeader.ts

```diff
@@ -57,13 +79,52 @@ export class AssessmentDetailsHeader {
 			}];
 
 		// create individual card component for each property in above list
-		headerCardsContainer.addItems(assessmentHeaderLabels.map(l => this.createCard(l)));
+		this._headerCardsContainer.addItems(assessmentHeaderLabels.map(l => this.createCard(l)));
 
-		headerContainer.addItems([targetTypeContainer, headerCardsContainer]);
+		headerContainer.addItems([this._targetTypeContainer, this._noTargetSelectedContainer, this._headerCardsContainer]);
 
 		return headerContainer;
 	}
 
+	// function creating ui for no target selected container.
+	private createTargetSelectedContainer(): azdata.FlexContainer {
```

Comment:
What happens when user click on save and opens the saved session? Do we show the previous selected target or no target?

Reply:
We show no taret screen only when migrationModel._targeType === undefined. In case of saved session, migrationModel._targeType must have stored value based on user selection, in that case er will show previous selected target only.

Reply:
Thanks. got it.